config/task_nav/how-to-drive-a-car.json

```diff
@@ -173,6 +173,27 @@
           }
         ]
       ]
-    }
+    },
+    "secondary_content": [
+      "/apply-for-your-full-driving-licence",
+      "/complain-about-a-driving-instructor",
+      "/contact-dvsa",
+      "/contact-the-dvla",
+      "/driving-eyesight-rules",
+      "/driving-test-cost",
+      "/dvlaforms",
+      "/government/publications/driving-instructor-grades-explained",
+      "/government/publications/know-your-traffic-signs",
+      "/guidance/rules-for-observing-driving-tests",
```

Comment:
"/government/publications/know-your-traffic-signs" and "/guidance/rules-for-observing-driving-tests",

Both don't show the content or sidebar